airgun/views/cloud_inventory.py

```diff
@@ -110,11 +111,18 @@ class CloudInventoryListView(BaseLoggedInView):
     obfuscate_hostnames = Switch('.//label[@for="rh-cloud-switcher-obfuscate_inventory_hostnames"]')
     obfuscate_ips = Switch('.//label[@for="rh-cloud-switcher-obfuscate_inventory_ips"]')
     exclude_packages = Switch('.//label[@for="rh-cloud-switcher-exclude_installed_packages"]')
-    cloud_connector = Pf4Button(locator='//button[normalize-space(.)="Configure cloud connector"]')
-    reconfigure_cloud_connector = Pf4Button(
+    auto_mismatch_deletion = Switch(
+        './/label[@for="rh-cloud-switcher-allow_auto_insights_mismatch_delete"]'
+    )
+
+    auto_upload_desc = PF5OUIAText('OUIA-Generated-Text-6')
+    manual_upload_desc = PF5OUIAText('OUIA-Generated-Text-7')
```

Comment:
I see that we use some these auto-generated IDs also elsewhere, but they can change over time, when some widgets get added or removed on the page.
If there is not a better locator, I'm fine with it, but we need to be aware that they don't change and then point to a different element.